src/main/kotlin/Actions/GetHTMLResponseAction.kt

```diff
@@ -0,0 +1,23 @@
+package Actions
+
+import httpstatus.HttpStatus
+import response.ResponseBuilder
+
+class GetHTMLResponseAction : Action {
+    private val statusCode = HttpStatus.OK
```

Comment:
Looks like you set the status to 200 by default. I think you don't need this.

~~what if you defaulted the `responseBuilder` to have a status of 200? I imagine 200 is the by far the most common status you will be setting, so it could default it.~~

Reply:
Yes, that is correct! I'll go ahead and remove the `statusCode` in the `Actions` where they aren't needed. Also, I am noticing I am calling `setStatusCode()`, passing in `statusCode = HttpStatus.OK`. I'll remove those as well since I've set the status to `200` by default in `HTTPResponseBuilder`